UI/EmuScreen.cpp

```diff
@@ -884,9 +884,9 @@ void EmuScreen::processAxis(const AxisInput &axis, int direction) {
 	KeyMap::AxisToPspButton(axis.deviceId, axis.axisId, -direction, &resultsOpposite);
 
 	int axisState = 0;
-	if ((direction == 1 && axis.value >= AXIS_BIND_THRESHOLD)) {
+	if (direction == 1 && (axis.value >= AXIS_BIND_THRESHOLD || g_Config.bMouseControl && axis.value >= 0.01)) {
```

Comment:
Although the parenthesis aren't technically needed here, people tend to misread statements that have mixed `||` and `&&` at the same level.  Normally, extra parenthesis would be best here.

But I'd suggest instead:
```c++
int axisState 0;
float threshold = g_Config.bMouseControl ? 0.01 : AXIS_BIND_THRESHOLD;
if (direction == 1 && axis.value >= threshold) {
```
(or perhaps even a `AXIS_BIND_TRESHOLD_MOUSE`.)

That should be the easiest to read and possibly more optimal anyway.

-[Unknown]

Reply:
Done :3